server/testdriver/core/src/main/java/org/infinispan/server/test/api/HotRodTestClientDriver.java

```diff
@@ -27,7 +27,7 @@ public HotRodTestClientDriver(TestServer testServer, TestClient testClient) {
       this.testClient = testClient;
 
       ConfigurationBuilder builder = new ConfigurationBuilder();
-      builder.maxRetries(1).connectionPool().maxActive(1);
+      builder.maxRetries(1).socketTimeout(10000).connectionPool().maxActive(1);
```

Comment:
@ryanemerson This looks like there's an issue here but I can't tell where and if it's on our side or DB2. But if the timeout isn't increased from default 2000ms then some of the test randomly fail on socket timeout

Reply:
I've seen it both Jenkins and locally

Reply:
I've seen this issue only with DB2

Reply:
I think the problem must be between the Infinispan server and DB2, with additional latencies (errors?) causing the client to timeout. I don't think we can justify increasing the socketTimeout for all tests just to satisfy the DB2 test cases.

We need to determine the root cause of the issue and once we know what it is, we need to either fix it or workaround by allowing the HR socketTimeout to be configurable via system properties so that we only increase it for the DB2 tests.

Reply:
So the timeouts are triggered when creating cache. The client sends the request to the server, which creates a table in the DB2, which takes time and results in socket timeout